Nondeterministic timeout in miner tests

```
ok  	github.com/NebulousLabs/Sia/modules/renter/contractor	0.008s

panic: test timed out after 5s

goroutine 38 [running]:

panic(0x859640, 0xc8203addd0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/runtime/panic.go:464 +0x3e6

testing.startAlarm.func1()

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/testing/testing.go:725 +0x14b

created by time.goFunc

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/time/sleep.go:129 +0x3a

goroutine 1 [chan receive]:

testing.RunTests(0xaf6380, 0xd6c060, 0xc, 0xc, 0xc82010e901)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/testing/testing.go:583 +0x8d2

testing.(*M).Run(0xc82004fef8, 0x4b9513)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/testing/testing.go:515 +0x81

main.main()

	github.com/NebulousLabs/Sia/modules/miner/_test/_testmain.go:76 +0x117

goroutine 17 [syscall, locked to thread]:

runtime.goexit()

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/runtime/asm_amd64.s:1998 +0x1

goroutine 28 [syscall]:

syscall.Syscall(0x4b, 0xe, 0x0, 0x0, 0xc8203cf000, 0x1000, 0x7fffffff)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/syscall/asm_linux_amd64.s:18 +0x5

syscall.Fdatasync(0xe, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/syscall/zsyscall_linux_amd64.go:472 +0x4a

github.com/NebulousLabs/bolt.fdatasync(0xc8202fa000, 0x0, 0x0)

	/home/travis/gopath/src/github.com/NebulousLabs/bolt/bolt_linux.go:9 +0x3f

github.com/NebulousLabs/bolt.(*Tx).write(0xc82039e2a0, 0x0, 0x0)

	/home/travis/gopath/src/github.com/NebulousLabs/bolt/tx.go:505 +0x425

github.com/NebulousLabs/bolt.(*Tx).Commit(0xc82039e2a0, 0x0, 0x0)

	/home/travis/gopath/src/github.com/NebulousLabs/bolt/tx.go:197 +0x512

github.com/NebulousLabs/bolt.(*DB).Update(0xc8202fa000, 0xc82018b5f8, 0x0, 0x0)

	/home/travis/gopath/src/github.com/NebulousLabs/bolt/db.go:580 +0x11d

github.com/NebulousLabs/Sia/modules/consensus.(*ConsensusSet).addBlockToTree(0xc820103800, 0x5fef67e41285e90e, 0x7b0b54148e69a0, 0x96019c33cc011008, 0x6727ef14625d9897, 0x0, 0x571f3b73, 0xc820377800, 0x1, 0x1, ...)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/consensus/accept.go:183 +0xf6

github.com/NebulousLabs/Sia/modules/consensus.(*ConsensusSet).managedAcceptBlock(0xc820103800, 0x5fef67e41285e90e, 0x7b0b54148e69a0, 0x96019c33cc011008, 0x6727ef14625d9897, 0x0, 0x571f3b73, 0xc820377800, 0x1, 0x1, ...)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/consensus/accept.go:259 +0x138

github.com/NebulousLabs/Sia/modules/consensus.(*ConsensusSet).AcceptBlock(0xc820103800, 0x5fef67e41285e90e, 0x7b0b54148e69a0, 0x96019c33cc011008, 0x6727ef14625d9897, 0x0, 0x571f3b73, 0xc820377800, 0x1, 0x1, ...)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/consensus/accept.go:286 +0x57

github.com/NebulousLabs/Sia/modules/miner.(*Miner).AddBlock(0xc8202036c0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/miner/testminer.go:49 +0x10e

github.com/NebulousLabs/Sia/modules/miner.createMinerTester(0xa7d7a0, 0x23, 0x0, 0x0, 0x0)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/miner/miner_test.go:87 +0xbef

github.com/NebulousLabs/Sia/modules/miner.TestIntegrationNilMinerDependencies(0xc8202ea360)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/miner/miner_test.go:131 +0x45

testing.tRunner(0xc8202ea360, 0xd6c0f0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/testing/testing.go:473 +0x98

created by testing.RunTests

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/testing/testing.go:582 +0x892

goroutine 12 [sleep]:

time.Sleep(0x4a817c800)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/runtime/time.go:59 +0xf9

github.com/NebulousLabs/Sia/sync.(*RWMutex).threadedDeadlockFinder(0xc82000be00)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/sync/lock.go:78 +0x5a1

created by github.com/NebulousLabs/Sia/sync.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/sync/lock.go:47 +0xbc

goroutine 15 [select]:

github.com/NebulousLabs/Sia/modules/gateway.(*Gateway).threadedPeerManager(0xc820110d20)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/peers.go:320 +0x343

created by github.com/NebulousLabs/Sia/modules/gateway.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/gateway.go:153 +0x89e

goroutine 16 [select]:

github.com/NebulousLabs/Sia/modules/gateway.(*Gateway).threadedNodeManager(0xc820110d20)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/nodes.go:132 +0x52f

created by github.com/NebulousLabs/Sia/modules/gateway.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/gateway.go:154 +0x8c3

goroutine 18 [IO wait]:

net.runtime_pollWait(0x2b1052105f50, 0x72, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/runtime/netpoll.go:160 +0x60

net.(*pollDesc).Wait(0xc820110ed0, 0x72, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/fd_poll_runtime.go:73 +0x3a

net.(*pollDesc).WaitRead(0xc820110ed0, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/fd_poll_runtime.go:78 +0x36

net.(*netFD).accept(0xc820110e70, 0x0, 0x2b1052140000, 0xc82017a080)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/fd_unix.go:426 +0x27c

net.(*TCPListener).AcceptTCP(0xc82002e138, 0x0, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/tcpsock_posix.go:254 +0x4d

net.(*TCPListener).Accept(0xc82002e138, 0x0, 0x0, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/tcpsock_posix.go:264 +0x3d

github.com/NebulousLabs/Sia/modules/gateway.(*Gateway).listen(0xc820110d20)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/peers.go:116 +0x41

created by github.com/NebulousLabs/Sia/modules/gateway.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/gateway.go:157 +0x8e8

goroutine 29 [sleep]:

time.Sleep(0x4a817c800)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/runtime/time.go:59 +0xf9

github.com/NebulousLabs/Sia/sync.(*RWMutex).threadedDeadlockFinder(0xc8202d7680)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/sync/lock.go:78 +0x5a1

created by github.com/NebulousLabs/Sia/sync.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/sync/lock.go:47 +0xbc

goroutine 32 [select]:

github.com/NebulousLabs/Sia/modules/gateway.(*Gateway).threadedPeerManager(0xc8202ec620)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/peers.go:320 +0x343

created by github.com/NebulousLabs/Sia/modules/gateway.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/gateway.go:153 +0x89e

goroutine 33 [select]:

github.com/NebulousLabs/Sia/modules/gateway.(*Gateway).threadedNodeManager(0xc8202ec620)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/nodes.go:132 +0x52f

created by github.com/NebulousLabs/Sia/modules/gateway.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/gateway.go:154 +0x8c3

goroutine 34 [IO wait]:

net.runtime_pollWait(0x2b1052105e90, 0x72, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/runtime/netpoll.go:160 +0x60

net.(*pollDesc).Wait(0xc8202ec7d0, 0x72, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/fd_poll_runtime.go:73 +0x3a

net.(*pollDesc).WaitRead(0xc8202ec7d0, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/fd_poll_runtime.go:78 +0x36

net.(*netFD).accept(0xc8202ec770, 0x0, 0x2b1052140000, 0xc82017adc0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/fd_unix.go:426 +0x27c

net.(*TCPListener).AcceptTCP(0xc82002ea70, 0x0, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/tcpsock_posix.go:254 +0x4d

net.(*TCPListener).Accept(0xc82002ea70, 0x0, 0x0, 0x0, 0x0)

	/home/travis/.gimme/versions/go1.6.linux.amd64/src/net/tcpsock_posix.go:264 +0x3d

github.com/NebulousLabs/Sia/modules/gateway.(*Gateway).listen(0xc8202ec620)

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/peers.go:116 +0x41

created by github.com/NebulousLabs/Sia/modules/gateway.New

	/home/travis/gopath/src/github.com/NebulousLabs/Sia/modules/gateway/gateway.go:157 +0x8e8

FAIL	github.com/NebulousLabs/Sia/modules/miner	6.410s
```

--- a/modules/miner/blockmanager_test.go
+++ b/modules/miner/blockmanager_test.go
@@ -14,7 +14,7 @@ import (
 func solveHeader(header types.BlockHeader, target types.Target) types.BlockHeader {
 	// Solve the header.
 	for {
-		// Incrememnt the nonce first to guarantee that a new header is formed
+		// Increment the nonce first to guarantee that a new header is formed
 		// - this helps check for pointer errors.
 		header.Nonce[0]++
 		id := crypto.HashObject(header)
@@ -44,7 +44,7 @@ func TestIntegrationHeaderForWork(t *testing.T) {
 	solvedHeader := solveHeader(header, target)
 	// Sanity check - header and solvedHeader should be different. (within the
 	// testing file, 'header' should always have a nonce of '0' and
-	// solvedHeader should never have a nonce of '0'.
+	// solvedHeader should never have a nonce of '0'.)
 	if header.Nonce == solvedHeader.Nonce {
 		t.Fatal("nonce memory is not independent")
 	}
@@ -63,7 +63,7 @@ func TestIntegrationHeaderForWorkUpdates(t *testing.T) {
 	if testing.Short() {
 		t.SkipNow()
 	}
-	mt, err := createMinerTester("TestIntegreationHeaderForWorkUpdates")
+	mt, err := createMinerTester("TestIntegrationHeaderForWorkUpdates")
 	if err != nil {
 		t.Fatal(err)
 	}
@@ -97,7 +97,7 @@ func TestIntegrationHeaderForWorkUpdates(t *testing.T) {
 	}
 }
 
-// TestIntegrationManyHeaders checks that requesting a full set of headers a
+// TestIntegrationManyHeaders checks that requesting a full set of headers in a
 // row results in all unique headers, and that all of them can be reassembled
 // into valid blocks.
 func TestIntegrationManyHeaders(t *testing.T) {

--- a/modules/miner/miner_test.go
+++ b/modules/miner/miner_test.go
@@ -128,6 +128,9 @@ func TestIntegrationMiner(t *testing.T) {
 // TestIntegrationNilMinerDependencies tests that the miner properly handles
 // nil inputs for its dependencies.
 func TestIntegrationNilMinerDependencies(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
 	mt, err := createMinerTester("TestIntegrationNilMinerDependencies")
 	if err != nil {
 		t.Fatal(err)
@@ -153,6 +156,9 @@ func TestIntegrationNilMinerDependencies(t *testing.T) {
 // TestIntegrationBlocksMined checks that the BlocksMined function correctly
 // indicates the number of real blocks and stale blocks that have been mined.
 func TestIntegrationBlocksMined(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+	}
 	mt, err := createMinerTester("TestIntegrationBlocksMined")
 	if err != nil {
 		t.Fatal(err)
@@ -226,7 +232,7 @@ func TestIntegrationBlocksMined(t *testing.T) {
 }
 
 // TestIntegrationAutoRescan triggers a rescan during a call to New and
-// verifies that the rescanning happens correctly. The rerscan is triggered by
+// verifies that the rescanning happens correctly. The rescan is triggered by
 // a call to New, instead of getting called directly.
 func TestIntegrationAutoRescan(t *testing.T) {
 	if testing.Short() {

